examples/api_handler_simple_server/server.py

```diff
@@ -0,0 +1,115 @@
+"""An example that shows how to use the API handler directly.
+
+For this to work with RemoteClient, the routes must match those expected
+by the client; i.e., /invoke, /batch, /stream, etc. No trailing slashes should be used.
+"""
+from importlib import metadata
+from typing import Annotated
+
+from fastapi import Depends, FastAPI, Request, Response
+from langchain_core.runnables import RunnableLambda
+from sse_starlette import EventSourceResponse
+
+from langserve import APIHandler
+
+PYDANTIC_VERSION = metadata.version("pydantic")
+_PYDANTIC_MAJOR_VERSION: int = int(PYDANTIC_VERSION.split(".")[0])
+
+app = FastAPI(
+    title="LangChain Server",
+    version="1.0",
+    description="Spin up a simple api server using Langchain's Runnable interfaces",
+)
+
+
+##
+# Example 1
+# Add an endpoint for invoke, and batch together
+def add_one(x: int) -> int:
+    """Add one to the given number."""
+    return x + 1
+
+
+chain = RunnableLambda(add_one)
+
+api_handler = APIHandler(chain, "/simple")
```

Comment:
Is there a more realistic scenario we can show this with?

Reply:
Yeah probably should!

I think another example though -- this example has complexity coming from the doc-generation itself.

Reply:
Should we name the example to make it more clear then it's about doc generation? I feel like people may not understand why this example exists / what it does that's special